app/templates/views/guidance/using-notify/unsubscribe-links.html

```diff
@@ -11,32 +12,36 @@
 
   <h1 class="heading-large">Unsubscribe links</h1>
 
-  <p class="govuk-body">You must let people opt out of receiving subscription emails.</p>
-  <p class="govuk-body">If you do not, recipients can report your emails as spam. If this happens we may stop you from using GOV.UK Notify.</p>
-  <p class="govuk-body">Transactional emails do not need to include an unsubscribe link.</p>
-   <p class="govuk-body">The GOV.​UK Service Manual explains the difference between <a class="govuk-link govuk-link--no-visited-state"
-    href="https://www.gov.uk/service-manual/design/sending-emails-and-text-messages#transactional-messages">transactional and subscription messages</a>.</p>
+  <p class="govuk-body">You must let people opt out of receiving subscription emails, otherwise they might report you for sending spam.</p>
+ <p class="govuk-body">With GOV.UK Notify you can:</p>
+<ul class="govuk-list govuk-list--bullet">
+    <li>add an unsubscribe link in your emails</li>
+    <li>get a list of the recipients who request to unsubscribe</li>
```

Comment:
```suggestion
    <li>get a list of the recipients who ask to unsubscribe</li>
```

I think ‘request’ maybe reads a little stiff and formal in this context.

‘Ask’ might not be right either – ‘want’ could also work. I’ll let you decide. Not a dealbreaker :-)

Reply:
I did think about this a lot! Because it's used throughout the UI (unsubscribe requests in the Dashboard) I think for consistency it's OK to use here. 
I definitely agree it's slightly formal but I want it to align with what a user will see in Notify.